tests/MainTest.elm

```diff
@@ -40,18 +40,20 @@ suite =
                     { "Email": { "type": "email" } },
                     { "Emails": { "type": "email" , "multiple": "true" } },
                     { "Digits": { "type": "text", "pattern": "^[0-9]+$" } },
+                    { "Nric2": { "inputTag": "nric-custom-ele", "attributes": { "type": "text", "pattern": "^[STGM][0-9]{7}[ABCDEFGHIZJ]$" } } },
                     { "Nric": { "type": "text", "pattern": "^[STGM][0-9]{7}[ABCDEFGHIZJ]$" } }
```

Comment:
old json is still supported, e.g.

```json
{
  "Nric": {
    "type": "text",
    "pattern": "^[STGM][0-9]{7}[ABCDEFGHIZJ]$"
  }
}
```

new json has `attributes` nesting, allowing for `inputTag` to be specified

```json
{
  "Nric2": {
    "inputTag": "nric-custom-ele",
    "attributes": {
      "type": "text",
      "pattern": "^[STGM][0-9]{7}[ABCDEFGHIZJ]$"
    }
  }
}
```